stable/elasticsearch-exporter/templates/servicemonitor.yaml

```diff
@@ -25,6 +25,10 @@ spec:
     port: {{ .Values.service.metricsPort.name }}
     path: {{ .Values.web.path }}
     scheme: {{ .Values.serviceMonitor.scheme }}
+    {{- if .Values.serviceMonitor.relabelings }}
+    relabelings:
+      {{- toYaml .Values.serviceMonitor.relabelings | nindent 6 }}
```

Comment:
would be good to have a consistent indentation (example https://github.com/helm/charts/blob/3b2db4aaf077516b74d2f3e379f4bc6a33b80d5b/stable/elasticsearch-exporter/templates/servicemonitor.yaml#L16)

Reply:
Done.